### PR TITLE
Add a file to keep track of track maintainers.

### PR DESCRIPTION
**Pull request by Insti**
https://github.com/exercism/discussions/issues/97 investigated the status of all Exercism language tracks. This file is intended to continue and maintain that process in a community driven manner.

My intention is for this version to be merged and then further PR's can fill out/update the rest of the information.

See "Ruby" under the "Maintained" section for an example of what information I think there should be for each track.

Ideally a bot could keep it updated, but until someone writes one we will just have to do it manually.

### CODE REVIEW

File: TRACK_MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+* F#
+* C#
+* Clojure
+* Objective-C
+
+### Maintained
+
+* Ruby   
+   Active: @kotp, @insti, @hilary, @tommyschaefer, @bmulvihill  
+   Alumni: @kytrynx, @bernardoamc, @Cohen-Carlisle
```
Comment (petertseng) [inline TRACK_MAINTAINERS.md]: you spelled her name wrong, is kytrinyx (unless there is in fact a secret kytrynx that I am not aware of)

File: TRACK_MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+### Inactive Tracks
+
+There has not been enough activity to generate a heartbeat for the following tracks:
+
+* Ceylon
+* ColdFusion
+* Haxe
+* Idris
+* The Netwide Assembler
+* Nim
```
Comment (Smarticles101) [inline TRACK_MAINTAINERS.md]: Should be Vim?

Comment (petertseng) [inline TRACK_MAINTAINERS.md]: Nim is for https://github.com/exercism/xnim (Nim should remain. If Vimscript is to be added, it should be added separately)

Comment (Smarticles101) [inline TRACK_MAINTAINERS.md]: Ah, I see now

File: TRACK_MAINTAINERS.md (hunk 1 above)
Comment (Insti) [inline TRACK_MAINTAINERS.md]: Thanks. 
How embarrassing :blush:

**Comment by petertseng** [comment]
for the purposes of this document, is there a need to make a distinction between orphaned and inactive?

**Comment by Insti** [comment]
> for the purposes of this document, is there a need to make a distinction between orphaned and inactive?

Probably not. Shall I merge the sections? Should it be under Orphaned or Inactive?

**Comment by petertseng** [comment]
> > for the purposes of this document, is there a need to make a distinction between orphaned and inactive?
> 
> Probably not. Shall I merge the sections? Should it be under Orphaned or Inactive?

I don't suppose I care so much, but if I were to suggest the path of least resistance, then I pick Orphaned, since that is the one that has the definition in the list (orphaned, endangered, etc.)

**Comment by Theodus** [comment]
I would consider the Pony track at risk, but not orphaned.

**Comment by Insti** [comment]
> I would consider the Pony track at risk, but not orphaned.

Can you submit a PR for this once the file is merged.
I'd rather not bog this PR down with making sure all the tracks are correct.

**Comment by Insti** [comment]
We probably should also link to the repositories.

**Comment by Insti** [comment]
> There does not seem to be much point for the @ preceding the names, since they do not get linked. Unless there is some per-user tool that will do it.

I was hoping Github would magically work.

**Comment by Theodus** [comment]
@Insti Sure, no problem

**Comment by Insti** [comment]
@Theodus it's merged now, you can submit your PR.

**Comment by Theodus** [comment]
@Insti Thanks!

**Pull request merged**